### PR TITLE
Ticket 372

### PR DESCRIPTION
**Pull request by mohitsengarr**
Changes for https://github.com/k8-proxy/cs-k8s-api/issues/372

### CODE REVIEW

**Comment by mohitsengarr** [comment]
Added Below Headers to the api response.

`X-SDK-Engine-Type` it's possible values are Eval and Prod.
`X-CDR-Platform`  it's possible values are Classic and Go.
`X-SDK-Engine-Expiry-Date` it's value would be expiry date of the eval SDK.

File: Source/Common/Glasswall.CloudProxy.Common/AdaptationService/AdaptationServiceResponse.cs [hunk 1]
```diff
@@ -18,6 +18,21 @@ public class AdaptationServiceResponse : IAdaptationServiceResponse
         public string MetaDataPresignedUrl { get; set; }
         public string SDKEngineVersion { get; set; }
         public RebuildProcessingStatus? RebuildProcessingStatus { get; set; }
+        public CDRPlatformType CDRPlatform
+        {
+            get
+            {
+                if (!string.IsNullOrWhiteSpace(SourcePresignedUrl))
```
Comment (ggrig) [inline Source/Common/Glasswall.CloudProxy.Common/AdaptationService/AdaptationServiceResponse.cs:25]: This is a little bit messy
Can't we have the values in config files?

**Review by ggrig — APPROVED** [review]

File: Source/Test/Glasswall.CS-SDK.UnitTests/DetailControllerTest.cs [hunk 2]
```diff
@@ -85,16 +111,26 @@ public void TestArgumentsOK()
         [TestMethod]
         public void TestGetVersionDetails()
```
Comment (ggrig) [inline Source/Test/Glasswall.CS-SDK.UnitTests/DetailControllerTest.cs:112]: This one fails in GitHub pipeline
Could you please fix it

**Review by ggrig — CHANGES_REQUESTED** [review]

File: Source/Common/Glasswall.CloudProxy.Common/AdaptationService/AdaptationServiceResponse.cs (hunk 1 above)
Comment (mohitsengarr) [reply Source/Common/Glasswall.CloudProxy.Common/AdaptationService/AdaptationServiceResponse.cs:25]: No, we can't.

File: Source/Test/Glasswall.CS-SDK.UnitTests/DetailControllerTest.cs (hunk 2 above)
Comment (mohitsengarr) [reply Source/Test/Glasswall.CS-SDK.UnitTests/DetailControllerTest.cs:112]: yes, Fixed

**Review by GINAGC — APPROVED** [review]

**Pull request merged**